integration/credential_revocation_concurrency_integration_test.go

```diff
@@ -111,17 +111,25 @@ func TestConcurrencyRevocationVerifiableCredentialIntegration(t *testing.T) {
 	assert.NotEmpty(t, schemaID)
 
 	const vcCount = 100
-	credStatusListIndexes := make([]string, vcCount)
+	credStatusListIndexes := make([]string, 0, vcCount)
```

Comment:
if 0 init does this syntax give us a benefit over `var credStatusListIndexes []string`?

Reply:
Yes, the correct capacity is allocated. More details [here](https://github.com/uber-go/guide/blob/master/style.md#specifying-slice-capacity)